src/helper/helper.js

```diff
@@ -44,16 +44,9 @@
  * @author bkuhn@google.com (Brian Kuhn)
  */
 
-goog.module('helper');
-
-const {type, hasOwn, isPlainObject} = goog.require('plain');
-
-/**
- * @define {boolean} When true, potential code errors will be logged to the
- * console. To enable this, run yarn build-debug to compile the distribution
- * code.
- */
-const DLH_DEBUG = goog.define('DLH_DEBUG', false);
+goog.module('datalayerhelper.helper.DataLayerHelper');
```

Comment:
I would change the name of the file too (to either data-layer-helper.js or data_layer_helper.js). This is Google convention, and can potentially be done in a separate PR.

Reply:
Switched it to data_layer_helper.js (an error was thrown when I tried data-layer-helper).

Reply:
I misunderstood this the first time to be namespaces. I made the namespaces upperCamelCase and the filename data_layer_helper.js